### DISCUSSION TITLE
How to use kia_uvo.set_charge_limits?

### DISCUSSION THREAD

**Discussion by creativethings** [Hyundai-Kia-Connect/kia_uvo#886] [Q&A]
Hi all,
I am testing the kia_uvo.set_charge_limits capability with my KIA CEED PHEV. However I am not sure how I can make it work. Calling the service does not change anything with my car. If I set either the AC or the DC value (which the GUI allows) I get the error I shoudl set both. If I do set both I get a general error. Does this service work at all? DOes anybody have a working example?
Many thanks!

**Comment by walter-ve** [comment]
For your info: setting the Charge limit on an EV6 works fine for me (although there seems a bug in the car that when you increase the limit it is not alway starting to charge).
